src/test/java/org/candlepin/async/tasks/CloudAccountOrgSetupJobTest.java

```diff
@@ -143,6 +143,39 @@ void ensureJobSuccessWithNewOrganization()
 
     }
 
+    @Test
+    void ensureJobSuccessWithNewNonAnonymousOrganization()
+        throws JobExecutionException, CouldNotAcquireCloudAccountLockException {
+        when(cloudReg.setupCloudAccountOrg(anyString(), anyString(), any(), anyString()))
+            .thenReturn(new CloudAccountData("owner_key", false));
+
+        CloudAccountOrgSetupJob regJob = new CloudAccountOrgSetupJob(cloudReg, ownerCurator);
+
+
+        String offering = TestUtil.randomString("offering");
+        CloudAccountOrgSetupJob.CloudAccountOrgSetupJobConfig jobConfig =
+            CloudAccountOrgSetupJob.createJobConfig()
+            .setCloudAccountId(TestUtil.randomString())
+            .setCloudOfferingId(offering)
+            .setCloudProvider(TestUtil.randomString())
+            .setOwnerKey(TestUtil.randomString());
+
+        AsyncJobStatus status = mock(AsyncJobStatus.class);
+        JobExecutionContext context = spy(new JobExecutionContext(status));
+        when(status.getJobArguments()).thenReturn(jobConfig.getJobArguments());
+
+        ArgumentCaptor<Object> captor = ArgumentCaptor.forClass(Object.class);
+
+        regJob.execute(context);
+
+        verify(context, times(1)).setJobResult(captor.capture());
```

Comment:
You can use just `verify(context)` here instead of adding the `times(1)` parameter

Reply:
Done